ts/Series/Pictorial/PictorialSeries.ts

```diff
@@ -0,0 +1,716 @@
+/* *
+ *
+ *  (c) 2010-2022 Torstein Honsi, Magdalena Gut
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import '../Column/ColumnSeries.js';
+import '../../Extensions/PatternFill.js';
+
+import type ColorType from '../../Core/Color/ColorType.js';
+import type ColumnSeriesType from '../Column/ColumnSeries';
+import type DataExtremesObject from '../../Core/Series/DataExtremesObject';
+import type PictorialSeriesOptions from './PictorialSeriesOptions';
+
+import A from '../../Core/Animation/AnimationUtilities.js';
+import Chart from '../../Core/Chart/Chart.js';
+import PictorialPoint from './PictorialPoint.js';
+import PictorialUtilities from './PictorialUtilities.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import StackItem from '../../Core/Axis/Stacking/StackItem.js';
+import SVGAttributes from '../../Core/Renderer/SVG/SVGAttributes.js';
+import U from '../../Core/Utilities.js';
+import { PictorialPathOptions } from './PictorialSeriesOptions';
+
+const ColumnSeries: typeof ColumnSeriesType = SeriesRegistry.seriesTypes.column;
+
+const {
+    animObject
+} = A;
+
+const {
+    getStackMetrics,
+    invertShadowGroup,
+    rescalePatternFill
+} = PictorialUtilities;
+
+const {
+    addEvent,
+    defined,
+    merge,
+    objectEach,
+    pick
+} = U;
+
+export interface StackShadowOptions {
+    borderColor?: ColorType;
+    borderWidth?: number;
+    color?: ColorType;
+    enabled?: boolean;
+}
+
+declare module '../../Core/Axis/AxisOptions' {
+    interface AxisOptions {
+        stackShadow?: StackShadowOptions;
+    }
+}
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * The pictorial series type.
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.pictorial
+ *
+ * @augments Highcharts.Series
+ */
+class PictorialSeries extends ColumnSeries {
+
+    /* *
+     *
+     *  Static Properties
+     *
+     * */
+
+    /**
+     * A pictorial chart uses vector images to represents the data.
+     * The shape of the data point is taken from the path parameter.
+     *
+     * @sample {highcharts} highcharts/demo/pictorial/
+     *         Pictorial chart
+     *
+     * @extends      plotOptions.column
+     * @since        next
+     * @product      highcharts
+     * @excluding    allAreas, borderRadius,
+     *               centerInCategory, colorAxis, connectEnds, connectNulls,
+     *               crisp, compare, compareBase, dataSorting, dashStyle,
+     *               dataAsColumns, linecap, lineWidth, shadow, onPoint
```

Comment:
Please, double-check this list too. For example `colorAxis` is disabled, then the `colorKey` option probably should be disabled too